saemref/map.jinja

```diff
@@ -11,7 +11,7 @@
 {% do default_settings.saemref.update(os_family_map) %}
 
 {% set saemref = salt['pillar.get'](
-        'saemref',
+        'saemref:lookup',
```

Comment:
I don't think this is the proper fix. Other formulas do not have the `:lookup` here.